packages/video_player/video_player/lib/video_player.dart

```diff
@@ -585,6 +605,56 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _videoPlayerPlatform.setVolume(_textureId, value.volume);
   }
 
+  /// Returns true if picture-in-picture is supported on the device.
+  Future<bool> isPictureInPictureSupported() =>
+      _videoPlayerPlatform.isPictureInPictureSupported();
+
+  /// Enables/disables to start picture-in-picture automatically when the app goes to the background.
+  Future<void> setAutomaticallyStartsPictureInPicture({
+    required bool enableStartPictureInPictureAutomaticallyFromInline,
+  }) async {
+    if (!value.isInitialized || _isDisposed) {
+      return;
+    }
+    await _videoPlayerPlatform.setAutomaticallyStartsPictureInPicture(
+      textureId: _textureId,
+      enableStartPictureInPictureAutomaticallyFromInline:
+          enableStartPictureInPictureAutomaticallyFromInline,
+    );
+  }
+
+  /// Sets the location of the video player view in order to animate the picture-in-picture view.
+  ///
+  /// The rect will represent the location of the video player view in Flutter. The rect will be
+  /// passed to the platform to position the native picture-in-picture overlay correctly.
```

Comment:
It's not clear to me what this last sentence is trying to communicate to a plugin client; it sounds like it's just saying that the internal implementation is correct, which isn't something we need to document. If this is trying to address my previous feedback, it should instead be saying something explicit about the coordinates/pixels this is in. E.g., "This should be provided in global Flutter coordinates using logic pixels." (If that's the case.)

Reply:
Also, this detail should move to the relevant property on `PictureInPictureOverlaySettings`, and this documentation should be more generic so that it doesn't assume things about the internals of the settings object.